test/e2e/run_test.go

```diff
@@ -368,16 +368,16 @@ USER bin`
 	})
 
 	It("podman run sysctl test", func() {
-		SkipIfRootless() // Network sysclts are not avalable root rootless
+		SkipIfRootless("Network sysclts are not avalable root rootless")
```

Comment:
typo (transposition): should be `sysctls`

Reply:
Fixed